### PR TITLE
LSR-28559 - Translate Sale Receipt Templates to Supported Languages

### PR DESCRIPTION
**Pull request by ls-jeremy-aalders**
This is just a reflection update to update our public facing repo with the translations that were done prior

### CODE REVIEW

**Comment by ls-jeremy-aalders** [comment]
This will also cover the other updates for 

'subtotal' on ES print templates
'total' on DE print templates.

File: receipt/SaleReceipt.tpl [hunk 1]
```diff
@@ -845,7 +901,7 @@ table.payments td.label {
 		{% endif %}
 
 		{% if Sale.Customer %}
-			{% if Sale.Customer.company|strlen > 0 %}
+			{% if Sale.Customer.company|strlen > 0 and not options.show_customer_name_only %}
```
Comment (ls-lindsay-powerharding) [inline receipt/SaleReceipt.tpl:904]: There seems to be some additional updates beyond what is required for this cycle, is that intended?

Comment (ls-jeremy-aalders) [reply receipt/SaleReceipt.tpl:904]: Will we need these changes to our public files? https://github.com/merchantos/webPOS/commit/a93700b6e037ed7629d7f2e95090159318baf29c

Comment (ls-lindsay-powerharding) [reply receipt/SaleReceipt.tpl:904]: Potentially, we should keep these templates the same as the source of truth, but for clarity sake let's keep this PR specific just to the changes required to show remote payment links for this cycle and we can create additional tasks to add in any other adjustments at another time

Comment (ls-jeremy-aalders) [reply receipt/SaleReceipt.tpl:904]: Since our public repos are suppose to match with what's in merchantOS, would it be possible to just keep the PR as is so that way we have both bases covered now and we don't have to come back?

File: receipt/SaleReceipt.tpl [hunk 2]
```diff
@@ -931,19 +987,19 @@ table.payments td.label {
...
-			{% if options.per_line_subtotal and options.discounted_line_items and Line.calcLineDiscount != 0 and not parameters.gift_receipt %}
-				<span class="strike">{{Line.unitQuantity}} x {{Line.unitPrice|money}}</span>
-			{% endif %}
-			{{Line.unitQuantity}}
-			{% if options.per_line_subtotal and not parameters.gift_receipt %} x
-				{% if options.discounted_line_items %}
-					{{ divide(Line.displayableSubtotal, Line.unitQuantity)|money }}
-				{% else %}
-					{{Line.displayableUnitPrice|money}}
+			<td data-automation="lineItemQuantity" class="quantity">
```
Comment (patrickpicher-lightspeed) [inline receipt/SaleReceipt.tpl:990]: thought: Indentation seems wrong, maybe revert these changes

File: receipt/SaleReceipt_fr_CA-QC.tpl [hunk 3]
```diff
@@ -227,10 +231,6 @@ table div.line_description_item_fee {
 	padding-left: 10px;
 }
 
-.footerSectionTitle + table div.line_description_item_fee {
-	padding-left: 25px;
-}
-
```
Comment (patrickpicher-lightspeed) [inline receipt/SaleReceipt_fr_CA-QC.tpl:233]: question: It seems this rule is declared on all receipt templates. Why are we removing it here?

Comment (ls-jeremy-aalders) [reply receipt/SaleReceipt_fr_CA-QC.tpl]: Not sure what happened here as to mark it for delete. It should now be updated

File: receipt/SaleReceipt.tpl (hunk 2 above)
Comment (ls-jeremy-aalders) [reply receipt/SaleReceipt.tpl]: Indenting should now be fixed.

**Review by patrickpicher-lightspeed — APPROVED** [review]
LGTM

**Review by ls-lindsay-powerharding — APPROVED** [review]
